common/src/main/java/io/quarkus/ts/openshift/common/util/AnnotationUtils.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.ts.openshift.common.util;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public final class AnnotationUtils {
```

Comment:
I had to create this utility because the AnnotatedElement.getAnnotationsByType only retrieves the annotations of the super class if the current class has no annotations.

Reply:
I know why I don't like inheritable annotations. BTW do you know what's wrong with your `getAllAnnotationsByType`? :-)

Reply:
Sorry, nothing's wrong, I got confused, I thought you're calling something from JUnit, but you're calling the method itself recursively. That's fine (though using a loop would probably be a bit more efficient :-) ).

(I still think inheritable annotations are harder to read, especially if we depend on order of repeatable annotations, which we do. But technically the code is fine.)

Reply:
I've removed the inheritable annotations and also removed the AnnotationUtils.